src/java/org/apache/cassandra/cql3/statements/schema/AlterTableStatement.java

```diff
@@ -553,6 +554,33 @@ public void validate(ClientState state)
             validateDefaultTimeToLive(attrs.asNewTableParams());
         }
 
+        private TableParams validateAndUpdateTransactionalMigration(TableParams prev, TableParams next)
+        {
+            boolean modeChange = prev.transactionalMode != next.transactionalMode;
+            boolean wasMigrating = prev.transactionalMigrationFrom != TransactionalMigrationFromMode.none;
+            boolean forceMigrationChange = prev.transactionalMigrationFrom != next.transactionalMigrationFrom;
```

Comment:
yes, or otherwise manually intervene

Reply:
I requested this elsewhere earlier, but can we log any time a forcible migration change happens since it's usually going to be some unsafe combination?